test/src/main/java/ai/timefold/solver/test/api/score/stream/SingleConstraintVerification.java

```diff
@@ -30,6 +30,6 @@ public interface SingleConstraintVerification<Solution_> {
     SingleConstraintAssertion given(@NonNull Object @NonNull... facts);
 
     @NonNull
-    SingleConstraintAssertion givenSolution(@NonNull Solution_ solution);
+    SingleConstraintListener givenSolution(@NonNull Solution_ solution);
```

Comment:
Dtto.

Reply:
`SingleConstraintListener` already extends `SingleConstraintAssertion`, and it should maintain backward compatibility.